HW9/Homework_9.py

```diff
@@ -0,0 +1,51 @@
+# 1. Fibonacci sequences using generator
+def fib_gen(n):
+    fibo_1, fibo_2 = 0, 1
+    for _ in range(1, n + 1):
```

Comment:
By default `range` starts from 0. You can change here it to `range(n)` and you will get the same number of iterations